lib/package-managers/abstract.package-manager.ts

```diff
@@ -23,7 +23,7 @@ export abstract class AbstractPackageManager {
     try {
       const commandArgs = `${this.cli.install} ${this.cli.silentFlag}`;
       const collect = true;
-      const dasherizedDirectory = dasherize(directory);
+      const dasherizedDirectory = normalizeToKebabOrSnakeCase(directory);
```

Comment:
```suggestion
      const normalizedDirectory = normalizeToKebabOrSnakeCase(directory);
```

(and then rename the occurrences below)

Reply:
Nice find, I checked the other places - this was the only variable rename required.
I've applied a fix and pushed it.